src/components/VBanner/VBanners.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <VMigrationNotice
+      v-if="shouldShowMigrationBanner"
+      @close="dismissBanner('cc-referral')"
+    />
+    <VTranslationStatusBanner
+      v-if="shouldShowTranslationBanner"
+      :banner-key="translationBannerId"
+      @close="dismissBanner(translationBannerId)"
+    />
+  </div>
+</template>
+<script lang="ts">
+import { computed, defineComponent } from '@nuxtjs/composition-api'
+
+import { useUiStore } from '~/stores/ui'
+import { TranslationBannerId } from '~/types/banners'
+
+export default defineComponent({
+  name: 'VBanners',
+  components: {
+    VMigrationNotice: () => import('~/components/VBanner/VMigrationNotice.vue'),
+    VTranslationStatusBanner: () =>
+      import('~/components/VBanner/VTranslationStatusBanner.vue'),
```

Comment:
It allows us to lazily load components. It's probably not as important for small components. We can also add the webpack comment to allow prefetching, although for the banners it doesn't make sense: the banner will only be displayed if the user goes to a language with fewer translated strings, but we would prefetch the code for everyone?

More info about it here:
https://vueschool.io/articles/vuejs-tutorials/lazy-loading-individual-vue-components-and-prefetching/

It would be nice to set up a way of measuring performance to see if things like this are really necessary or not.